resident/resident-service/src/main/java/io/mosip/resident/service/impl/ResidentServiceImpl.java

```diff
@@ -1541,17 +1543,15 @@ public byte[] downloadCard(String eventId) {
 			if (residentTransactionEntity.isPresent()) {
 				String requestTypeCode = residentTransactionEntity.get().getRequestTypeCode();
 				RequestType requestType = RequestType.getRequestTypeFromString(requestTypeCode);
-				if (requestType.name().equalsIgnoreCase(RequestType.UPDATE_MY_UIN.name())) {
+				if (requestType.equals(RequestType.UPDATE_MY_UIN)) {
 					cardType = IdType.UIN.name();
```

Comment:
Is the cardType variable used anywhere?

Reply:
yes, it is used to create the file name (getFileName()) for card...whether it's UIN card or VID card.